torch/testing/_internal/common_methods_invocations.py

```diff
@@ -18354,6 +18372,14 @@ def __init__(
     ElementwiseUnaryPythonRefInfo(
         "_refs.log",
         torch_opinfo_name="log",
+        skips=(
+            # RuntimeError: "abs_cuda" not implemented for 'ComplexHalf'
```

Comment:
we should remove these skips after https://github.com/pytorch/pytorch/pull/77446 is merged